From c986ed981ae6a622a453c533389994b6aed6359b Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 26 May 2021 17:59:56 +0900
Subject: [PATCH] cifsd: remove unnecessary parentheses around
Git-commit: c986ed981ae6a622a453c533389994b6aed6359b
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Fix warnings from checkpatch.pl --strict :

 CHECK: Unnecessary parentheses around 'brk_op->o_lease->new_state ==
 SMB2_LEASE_NONE_LE'
 #1511: FILE: oplock.c:1511:
 +               if (brk_op->is_lease &&
 +                   (brk_op->o_lease->new_state == SMB2_LEASE_NONE_LE)
 &&
 +                   atomic_read(&brk_op->breaking_cnt))

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/oplock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/cifsd/oplock.c b/fs/cifsd/oplock.c
index 56c68e9cb7ff..f76de7861e7b 100644
--- a/fs/cifsd/oplock.c
+++ b/fs/cifsd/oplock.c
@@ -1262,7 +1262,7 @@ void smb_break_all_levII_oplock(struct ksmbd_work *work, struct ksmbd_file *fp,
 
 		/* Skip oplock being break to none */
 		if (brk_op->is_lease &&
-		    (brk_op->o_lease->new_state == SMB2_LEASE_NONE_LE) &&
+		    brk_op->o_lease->new_state == SMB2_LEASE_NONE_LE &&
 		    atomic_read(&brk_op->breaking_cnt))
 			goto next;
 
-- 
2.33.0

